example-values/resources.yaml

```diff
@@ -136,6 +136,120 @@ ml-model-service:
       limits:
         memory: 3Gi
         cpu: 1500m
+  argo:
+    ui:
+      resources:
+        requests:
+          memory: "128Mi"
+          cpu: "50m"
+        limits:
+          memory: "256Mi"
+          cpu: "100m"
+    controller:
+      resources:
+        requests:
+          memory: "128Mi"
+          cpu: "50m"
+        limits:
+          memory: "256Mi"
+          cpu: "100m"
+
+  ambassador:
+    resources:
+      requests:
+        memory: "128Mi"
+        cpu: "100m"
+      limits:
+        memory: "256Mi"
+        cpu: "200m"
+
+fusion-question-answering:
+  resources:
+    default:
+      requests:
+        memory: "64Mi"
+        cpu: "10m"
+      limits:
+        memory: "100Mi"
+        cpu: "30m"
+
+    coldstart:      
+      dataPuller: 
+        requests:
+          memory: "2Gi" 
+          cpu: "100m"
+        limits:
+          memory: "4Gi"
+          cpu: "500m"
+      train: 
+        requests:
+          memory: "6Gi"
+          cpu: "10000m"
+        limits:
+          memory: "8Gi"
+          cpu: "20000m"
+      seldon:
+        requests:
+          memory: "2Gi"
+          cpu: "200m"
+        limits:
+          memory: "3Gi"
+          cpu: "1000m"
+
+    supervised:
+      dataPuller: 
+        requests:
+          memory: "2Gi" 
+          cpu: "100m"
+        limits:
+          memory: "4Gi"
+          cpu: "500m"
+      train: 
+        requests:
+          memory: "8Gi"
+          cpu: "10000m"
+        limits:
+          memory: "10Gi"
+          cpu: "20000m"
```

Comment:
20 cores?

Reply:
That's a slip, it should be 10. I'll fix.

Reply:
Did some more testing on this and I think my first stab at the limits may have fallen victim to pod evictions rather than OOM errors. I managed to get it running with 4 cores, albeit twice as slow as a non-limited run. I've commented out the limits and added a note that it requires 5.1.2